### DISCUSSION TITLE
Triangles mania editing

### DISCUSSION THREAD

**Discussion by Redd56** [ppy/osu#21302] [Q&A]
i personaly use the triangles skin in mania (the old default on lazer) and am starting to get into more keys than just 4, but wanted to edit the skin, unfotunately it cant be exported and i was wondering if it would be possible to make osk's either export default skin in entirety for editing or if the skin itself is in fact the cs files i found for it to be supported to have code based ones avalible to skinners in osk files.(and be able to export triangles, i genuinely love it)

**Comment by Redd56** [comment]
if the skin is actualy just somewhere in the repo and i missed it feel free to point me to it

**Reply by ghost** [reply]
both triangle and argon skins are generated using code, for all rulesets, it that's what you're asking

**Reply by Redd56** [reply]
guess i gotta go make a fork or somehow have someone replicate them

**Reply by mk56-spn** [reply]
A word of warning. As loma mentioned the skin is hardcoded into osu. Meaning even if you are successful in making edits to it through code the OSU build you make will not be able to connect to the main Lazer server

**Reply by Redd56** [reply]
:skull:
